lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -41,6 +44,11 @@ def lookup_in_env(env_variable_name, project_name=nil, default=nil)
           project_name && ENV[project_specific_name] || ENV[env_variable_name] || default
         end
 
+        # @return [Boolean] Whether Puppet's internal builds are accessible from here
+        def dev_builds_accessible?
```

Comment:
As this is defined right now, it will return true for the scenario where localhost has access to the builds URL, but the test host (VM) doesn't have access to it (since [link_exists? uses Net::HTTP](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/helpers/web_helpers.rb#L15-L31)). Is that desirable?

Reply:
Yeah, this is more of a thing to just determine whether the user is on the VPN. I'm not aware of any way that a beaker SUT could access builds.delivery when the control host couldn't, but I do think it makes sense to do this on the SUT instead. I'll update it